### PR TITLE
add ability to parse celery task from log line

### PR DESCRIPTION
**Pull request by snopoke**
Build on https://github.com/dimagi/datadog-parsers/pull/38 to add support for parsing celery task name from the log lines.

See https://github.com/dimagi/commcare-hq/pull/29542/

### CODE REVIEW

File: tests/test_couch/test_couch_parsers.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 logging.basicConfig(level=logging.DEBUG)
 
 SIMPLE = '2015-10-31 18:32:03,963 [:mvp-pampaida] /a/mvp-pampaida/receiver/630916e49084b142c0a5a69c3a52b9b3/ PUT None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+URL_NO_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] /a/mvp-pampaida/receiver/630916e49084b142c0a5a69c3a52b9b3/ - commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+NO_URL_NO_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] - - commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
+NO_URL_TASK = '2015-10-31 18:32:03,963 [:mvp-pampaida] - corehq.apps.tasks.build_app commcarehq PUT 200 None d3abf611f2acdc7b4c32f7ebf4982a88 0:00:00.191515'
```
Comment (dannyroberts) [inline tests/test_couch/test_couch_parsers.py:12]: nit: an http method (like `PUT` here) wouldn't show up for a celery task, right? Not sure if that matters, but wanted to make sure the test is representative

**Review by dannyroberts — APPROVED** [review]
Awesome!

File: tests/test_couch/test_couch_parsers.py (hunk 1 above)
Comment (snopoke) [reply tests/test_couch/test_couch_parsers.py:12]: why not? this is the http method of the couch request, not the HQ request.

Comment (dannyroberts) [reply tests/test_couch/test_couch_parsers.py:12]: Oh my, sorry I just woke up. Yeah that makes total sense.

**Pull request merged**